### ISSUE TITLE
Providing option to select specific nifty index stocks.

### ISSUE THREAD

**Issue by swarpatel23** [pranjal-joshi/Screeni-py#59]
Currently, we are screening all NSE stocks. we can provide an option to select specific nifty index stocks.

![image](https://user-images.githubusercontent.com/47181082/122644948-5ab1b680-d135-11eb-885b-11e6e7176dab.png)

As shown in the above image.

Can I make PR ?

**Comment by pranjal-joshi** [comment]
Sure. Let's do it!

**Comment by pranjal-joshi** [comment]
I am getting a following error while using the screener from the machine sitting behind the proxy authentication.
```
[+] Select option: 2
[+] Getting Stock Codes From NSE... **check_hostname requires server_hostname**
=> Error getting stock codes from NSE! Press any key to exit!
```
For downloading CSVs from the NSE, the request should consider using the proxy auth if available [like this](https://github.com/pranjal-joshi/Screeni-py/blob/23f0c4210399622f1353cbbeb65cb94ba909ee16/src/screenipy.py#L51)

**Comment by pranjal-joshi** [comment]
Proxy auth-related issue fixed in ed8103e.
